doc/devel/index.rst

```diff
@@ -94,29 +99,23 @@ New contributors
             :link-type: ref
             :shadow: none
 
-            :octicon:`code;1em;sd-text-info` Contribute code
+            :octicon:`code;1em;sd-text-info` Code
```

Comment:
I feel just the nouns are too terse. What do the mean? Read documentation, write documentation? Documentation of the project or documentation on contributing?

"Contribute code" and "Write documentation" weren't too bad. "Join the community" could be the third one. Alternatively, "Work on code", "Work on documentation", "Work with the community".